frontend/src/v4/routes/viewerGui/components/risks/risks.component.tsx

```diff
@@ -87,14 +87,6 @@ export class Risks extends PureComponent<IProps, any> {
 		return getHeaderMenuItems(this.props);
 	}
 
-	get showDefaultHiddenItems() {
-		if (this.props.selectedFilters.length) {
-			return this.props.selectedFilters
-				.some(({ value: { value } }) => RISK_DEFAULT_HIDDEN_LEVELS.includes(value));
-		}
-		return false;
-	}
-
 	public renderDetailsView = renderWhenTrue(() => (
```

Comment:
I see difference between the default filters in the original issue #4584 
![image](https://github.com/3drepo/3drepo.io/assets/11945337/99a3e902-d9fc-401a-ad31-d289e9ef8445)
and in the PR:

<img width="418" alt="image" src="https://github.com/3drepo/3drepo.io/assets/42032433/a3fb4717-0ac6-4f87-8612-dfdd77703bf5">

Reply:
same as the previous comment